Examples/Framework/include/ActsExamples/EventData/DigitizedHit.hpp

```diff
@@ -0,0 +1,48 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Surfaces/Surface.hpp"
+#include "ActsExamples/EventData/GeometryContainers.hpp"
+#include "ActsExamples/EventData/SimHit.hpp"
+
+namespace ActsExamples {
+
+/// @brief DigitizedHit class that acts as a source link
+/// in the `FittableMeasurement` for further usage
+///
+/// This class thus needs to comply with the `SourceLinkConcept`
+class DigitizedHit {
```

Comment:
Since this is a `SourceLink` consider renaming it e.g. to `DigitizationSourceLink`.

Reply:
I'd rather keep it, yes, it is a `SourceLink` but it would also make sense as a standalone entity even if not within the context of a measurement.

Reply:
Ok, I can understand that. We leave it as-is.